src/igvfd/schemas/changelogs/document.md

```diff
@@ -0,0 +1,26 @@
+## Changelog for document.json
+
+### Minor changes since schema version 1
+
+* *document_type* property enum list was expanded to include:
+    "enum": [
+        "characterization",
+        "computational protocol",
+        "experimental protocol",
+        "file format specification",
+        "image",
+        "plasmid map",
+        "standards"
+    ]
+* *characterization_method* property was added:
+    "enum": [
+        "FACS",
+        "immunoblot",
+        "immunofluorescence",
+        "immunoprecipitation",
+        "mass spectrometry",
+        "PCR",
+        "restriction digest",
+        "RT-qPCR",
+        "Sequencing"
```

Comment:
To be consistent with the other enums, "sequencing" could be lower case.

Reply:
ok